projects/core/public_api.ts

```diff
@@ -5,3 +5,4 @@
 export * from './src/config/index';
 export * from './src/site-context/index';
 export * from './src/occ/index';
+export * from './src/configurable-routes/index';
```

Comment:
Shall we move the configurable-routes under a route folder? Then, when we merge the routing logic from the stoerfrontlib, we have one folder that contains both.

Reply:
👍